SCons/Tool/MSCommon/vc.py

```diff
@@ -942,8 +942,9 @@ def msvc_setup_env(env):
         use_script = use_script.strip()
         if not os.path.exists(use_script):
             raise MSVCScriptNotFound('Script specified by MSVC_USE_SCRIPT not found: "{}"'.format(use_script))
-        debug('use_script 1 %s', repr(use_script))
-        d = script_env(use_script)
+        args = env.get('MSVC_SCRIPT_ARGS', None)
```

Comment:
Please use env.subst("$MSVC_SCRIPT_ARGS") instead of get. That let's evaluation happen.